src/material.cpp

```diff
@@ -142,17 +137,12 @@ Entry* probe(const Position& pos) {
   if (pos.this_thread()->endgames.probe(key, e->evaluationFunction))
       return e;
 
-  if (is_KXK<WHITE>(pos))
-  {
-      e->evaluationFunction = &EvaluateKXK[WHITE];
-      return e;
-  }
-
-  if (is_KXK<BLACK>(pos))
-  {
-      e->evaluationFunction = &EvaluateKXK[BLACK];
-      return e;
-  }
+  for (Color c = WHITE; c <= BLACK; ++c)
```

Comment:
Here is better to remove the loop and write

 if (is_KXK(pos, WHITE) ||  is_KXK(pos, BLACK))